modules/custom/d_p_carousel/d_p_carousel.module

```diff
@@ -52,9 +52,6 @@ function d_p_carousel_preprocess_field(&$variables) {
           if ($num_items === 1) {
             $columns = 1;
           }
-          elseif ($num_items <= $columns) {
```

Comment:
Czy tutaj nie zepsują się domyślne zachowania karuzeli dla dotychczasowych userów drooplera?
To było zabezpieczenie żeby itemy nie pokazywały się niesymetrycznie.

Reply:
no właśnie

Reply:
to musze pomyslec nad tym jak zrobic, zeby na desktopie nie pokazywal sie slider...